tests/primeqa/text_classification/processors/postprocessors/test_text_classifier_postprocessor.py

```diff
@@ -0,0 +1,98 @@
+from datasets import Dataset
+import pytest
+from pytest import raises
+import json
+
+import numpy as np
+
+from primeqa.text_classification.processors.postprocessors.text_classifier import TextClassifierPostProcessor
+#from primeqa.boolqa.processors.postprocessors.boolqa_classifier import BoolQAClassifierPostProcessor
+
+from tests.primeqa.mrc.common.base import UnitTest
+
+
+class TestExtractivePostProcessor(UnitTest):
+    _predict_scores_for_get_pred=np.array([[-4.8621945,  7.0225224, -1.3763661],
+       [ 5.729403 , -1.9911858, -3.4178042],
+       [-4.871264 ,  2.7192504,  2.9824948]])
+
+    # we use rounded scores here to not trip on float comparison - change the reference files by hand!
+    _predict_scores_for_process=np.array([[ -4.0, -3.4, 7.7 ] ,
+                                     [ -4.3, -2.9, 7.4 ]])    
+
+    _expected_predictions0=np.array([1, 0, 2])
+    _expected_predictions1=np.array([2, 0, 2])
+
+    _examples = Dataset.from_json('tests/resources/text_classification/processors/postprocessors/examples.json')
+    _features = Dataset.from_json('tests/resources/text_classification/processors/postprocessors/features.json')
+    _expected_references=json.load(open('tests/resources/text_classification/processors/postprocessors/references.json'))
+    _expected_predictions=json.load(open('tests/resources/text_classification/processors/postprocessors/predictions.json'))
+    _expected_processed_predictions=json.load(open('tests/resources/text_classification/processors/postprocessors/processed_predictions.json'))
+    _expected_processed_predictions_no_ref=json.load(open('tests/resources/text_classification/processors/postprocessors/processed_predictions_no_ref.json'))
+    #------------------------------------------------------
+    # sample data obtained with this embedded in process_references_and_predictions()
```

Comment:
Delete commented lines if it is not needed.

Reply:
this comment may be useful if future updates are needed